src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java

```diff
@@ -100,6 +100,16 @@ protected ActionResult execute() {
             data.emailToEdit.emailId = emailId;
         }
 
+        if (addressModeOn && groupModeOn) {
+            statusToAdmin = "Email queued for sending." + "<br/>" + "Receipient: " + addressReceiverListString
+                            + "<br/>" + "Group receiver's list " + groupReceiverListFileKey;
+            statusToUser.clear();
+
+            statusToUser.add(new StatusMessage("Email will be sent within an hour to "
```

Comment:
@xpdavid @Neurrone Then should I make the changes, so that it shows two messages to the user.
Just a suggestion showing only one message feels to me like a more mature UI
![emaill](https://cloud.githubusercontent.com/assets/17481466/25038069/513eb1e0-211a-11e7-87ae-ca0509946482.PNG)
![email-2](https://cloud.githubusercontent.com/assets/17481466/25038073/552ac33e-211a-11e7-869f-0d4257e7c6db.PNG)